src/Domain/Common/EntityEnums/FieldType.php

```diff
@@ -7,28 +7,28 @@
 // maps to Doctrine\DBAL\Types\Types to enum
 enum FieldType: string
 {
-    case ASCII_STRING         = Types::ASCII_STRING;
-    case BIGINT               = Types::BIGINT;
-    case BINARY               = Types::BINARY;
-    case BLOB                 = Types::BLOB;
-    case BOOLEAN              = Types::BOOLEAN;
-    case DATE_MUTABLE         = Types::DATE_MUTABLE;
-    case DATE_IMMUTABLE       = Types::DATE_IMMUTABLE;
-    case DATEINTERVAL         = Types::DATEINTERVAL;
-    case DATETIME_MUTABLE     = Types::DATETIME_MUTABLE;
-    case DATETIME_IMMUTABLE   = Types::DATETIME_IMMUTABLE;
-    case DATETIMETZ_MUTABLE   = Types::DATETIMETZ_MUTABLE;
-    case DATETIMETZ_IMMUTABLE = Types::DATETIMETZ_IMMUTABLE;
-    case DECIMAL              = Types::DECIMAL;
-    case FLOAT                = Types::FLOAT;
-    case GUID                 = Types::GUID;
-    case INTEGER              = Types::INTEGER;
-    case JSON                 = Types::JSON;
+    case ASCII_STRING         = 'ascii_string'; //Types::ASCII_STRING
```

Comment:
somehow phpstan start complaining :
Child process error (exit code 255): Fatal error: Enum case value must be compile-time evaluatable

so just copied the value

Reply:
yes I saw that too, but only after the phpstan on GitHub (so not locally for me), so I was confused about that. Thanks for finding the fix